sympy/tensor/indexed.py

```diff
@@ -1,115 +1,10 @@
-"""Module that defines indexed objects
```

Comment:
It's usually better to keep docs in the code. People working on the code tend come here by reading code, and they might not be aware that anything interesting is in the docs and miss the introduction.

The preferred approaches would be:
* Find a piece of code that's common to all places where the docstring would be appropriate but redundant. In 
Approaches that might work:
* Move the docs into a common superclass, or maybe into `tensor`'s `__init__.py` (not sure if the latter will actually work).
* Leave the docs where they are, and reference them from the other module.